ibis/expr/types/relations.py

```diff
@@ -3611,6 +3611,189 @@ def pivot_wider(
 
         return self.group_by(id_cols).aggregate(**aggs)
 
+    def relocate(
+        self,
+        *selectors: str | s.Selector,
+        before: str | s.Selector | None = None,
+        after: str | s.Selector | None = None,
+        **kwargs: str | s.Selector,
+    ) -> Table:
+        """Relocate columns in a table.
+
+        Parameters
+        ----------
+        selectors
+            Selectors to relocate.
```

Comment:
What do you think about naming this argument `columns` instead of selectors? My initial expectation on seeing an arg named `selectors` was that this probably doesn't support column names directly. We already have several places where selectors can be passed in where columns are supported (making that use case more expected). Calling it `columns` also makes describing the function easier ("relocate takes one or more columns (or selections of columns) and relocates them before or after other specified columns").

If you want to keep the name as is, adding `column name or selector` here as you've done below for `before`/`after` would help.

Reply:
`columns` makes sense to me